src/content/app/regulatory-activity-viewer/components/activity-viewer-popup/GenePopupContent.tsx

```diff
@@ -0,0 +1,81 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { getStrandDisplayName } from 'src/shared/helpers/formatters/strandFormatter';
+import { getFormattedLocation } from 'src/shared/helpers/formatters/regionFormatter';
+
+import TextButton from 'src/shared/components/text-button/TextButton';
+
+import type { GeneInRegionOverview } from 'src/content/app/regulatory-activity-viewer/types/regionOverview';
+
+import styles from './AcrivityViewerPopup.module.css';
+
+type GeneField =
+  | 'stable_id'
+  | 'symbol'
+  | 'unversioned_stable_id'
+  | 'biotype'
+  | 'strand'
+  | 'start'
+  | 'end';
+
+type Props = {
+  gene: Pick<GeneInRegionOverview, GeneField> & { region_name: string };
+  onFocus: () => void;
+};
+
+const GenePopupContent = (props: Props) => {
+  const { gene } = props;
+
+  const geneSymbolAndIdentifier = gene.symbol ? (
+    <>
+      <span>{gene.symbol}</span> <span>{gene.stable_id}</span>
+    </>
+  ) : (
+    <span>{gene.stable_id}</span>
+  );
+
+  return (
+    <div>
+      <div className={styles.regularRow}>
+        <span className={styles.light}>Gene </span>
+        {geneSymbolAndIdentifier}
+      </div>
+      <div className={styles.regularRow}>
+        <span className={styles.light}>Biotype </span>
+        <span>{gene.biotype}</span>
+      </div>
+      <div className={styles.light}>
+        <span className={styles.strand}>
+          {/* TODO: Change Strand enum into a union type */}
+          {getStrandDisplayName(gene.strand as any)}{' '}
```

Comment:
Because the type of gene.strand is a union of two strings: `"forward" | "reverse"`, whereas the `getStrandDisplayName` function was written when we used enums, and expects an `enum Strand { FORWARD: "forward", REVERSE: "reverse" }. The actual value that is passed into the function is the same (a string "forward" or "reverse"); but typescript refuses to accept a string union where a parameter was typed as an enum.

I think we should just gradually get rid of all our enums, and Strand is a good candidate for being the first to go.